src/System.CommandLine/Invocation/TypoCorrection.cs

```diff
@@ -7,29 +7,21 @@
 
 namespace System.CommandLine.Invocation
 {
-    internal class TypoCorrection
+    internal static class TypoCorrection
     {
-        private readonly int _maxLevenshteinDistance;
-
-        public TypoCorrection(int maxLevenshteinDistance)
+        internal static void ProvideSuggestions(InvocationContext context)
```

Comment:
InvocationContext provides all we need, the class and method can be static now (fewer allocs)